docs/install/install-linux.md

```diff
@@ -0,0 +1,31 @@
+# Linux Installation Guide
+
+There are a number of ways to chose from to install Vignette on Linux:
+
+- [Steam](#Steam)
+
+- [AUR](#AUR) (on Arch Linux)
```

Comment:
I don't think we have to specify that the AUR is only on Arch... technically everyone can use it (although not officially supported), even on Debian...

```suggestion
- [AUR](#AUR)
```